app/workers/follows/update_points_worker.rb

```diff
@@ -34,7 +35,8 @@ def calculate_implicit_points(tag, user)
       tags = articles.pluck(:cached_tag_list).flat_map { |list| list.split(", ") }
       occurrences = tags.count(tag.name)
       bonus = inverse_popularity_bonus(tag)
-      Math.log(occurrences + bonus + 1) # +1 is purelt to avoid log(0) => -infinity
+      finalized_points(occurrences, bonus, user)
+      # Math.log(occurrences + bonus + 1) # +1 is purelt to avoid log(0) => -infinity
```

Comment:
I think we can remove this line and if you want add a comment below to the "base" explaining that's our existing implementation.